docs/_quarto.yml

```diff
@@ -111,6 +111,7 @@ website:
       collapse-level: 2
       contents:
         - support_matrix.qmd
+        - reference/connection.qmd
```

Comment:
Can you move this to be only in the API docs? I feel like we should keep the backend tab to only contain the support matrix and the backends and nothing else. Otherwise it looks cluttered to me.

Reply:
Sure. Under what section and what should the page be named?

Reply:
Can you add it this in a follow up PR? In that PR I think you remove this section and let the API docs show up as they usually do under the `Reference` tab.